client/redux/user/types.ts

```diff
@@ -1,4 +1,4 @@
-import { USER_ROLE } from '~/../shared/SharedConstants';
+// import { UserRole } from '~/../shared/SharedConstants';
```

Comment:
You changed then commented it. If it's not needed, delete it.

Reply:
I see you resolved this conversation but the code is still there. You should delete it if it's not needed